CHANGES.rst

```diff
@@ -19,6 +19,7 @@ Bugfixes
   timeline (:pr:`5218`)
 - Fix incorrect date in multi-day meeting date selector dropdown in certain timezones
   (:pr:`5223`)
+- Add support for abstract cloning (:pr:`5217`)
```

Comment:
that's not really a bugfix but rather an improvement :P

Reply:
My bad, this was auto-merged